src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -103,14 +103,19 @@ class ComService(
       body.getSortBy(),
     )
 
-    val resultsWithLicences: List<Pair<CaseloadResult, Licence?>> =
+    val prisonersPlusLicencesList: List<Pair<CaseloadResult, Licence?>> =
```

Comment:
CaseLoadResults are the result of a search against probation, we often call these offenders rather than prisoners (not everyone in probation is in prison) so this is slightly confusing name.